lib/osf-components/addon/components/sort-button/styles.scss

```diff
@@ -23,12 +23,12 @@
             box-shadow: none;
         }
     }
-}
 
-.selected {
-    color: $color-text-blue-dark !important;
-}
+    :global(.selected) {
+        color: $color-text-blue-dark !important;
+    }
 
-.not-selected {
-    color: $color-text-gray-dark !important;
+    :global(.not-selected) {
+        color: $color-text-gray-dark !important;
+    }
```

Comment:
Yeah, this'll work. I really wish we had better consistency in design... (but that's not your problem :wink:)